pkgs/development/libraries/libutempter/default.nix

```diff
@@ -4,17 +4,19 @@ with lib;
 
 stdenv.mkDerivation rec {
   pname = "libutempter";
-  version = "1.1.6";
+  version = "1.2.1";
 
   src = fetchurl {
-    url = "mirror://ubuntu/pool/main/libu/libutempter/libutempter_${version}.orig.tar.bz2";
-    sha256 = "15y3xbgznjxnfmix4xg3bwmqdvghdw7slbhazb0ybmyf65gmd65q";
+    url = "http://ftp.altlinux.org/pub/people/ldv/utempter/libutempter-${version}.tar.gz";
+    sha256 = "sha256-ln/vNy85HeUBhDrYdXDGz12r2WUfAPF4MJD7wSsqNMs=";
```

Comment:
nit: `hash` would be ideal here instead of `sha256` but that's really minor

Reply:
when we decide to use hash instead of sha256 for all things we can just `sd 'sha256 = "sha256-' 'hash = "sha256-' $(fd --type f)`